### DISCUSSION TITLE
PhotoPrism Not Indexing All Photos

### DISCUSSION THREAD

**Discussion by zollan23** [photoprism/photoprism#2815] [Q&A]
I have installed PP on an Unraid server using Maria DB. I have mapped the originals folder to my "Photos" share where all of my digital images live (over 50,000). I have both dng and jpg files in this share and have used the ppignore file to ignore dng files and NEF files (any raw images). 

After running the scan and indexing the photos, I'm noticing it has only picked up about 11k photos and several of the photos are missing. My basic photos folder structure is: Photos > Camera > Year > Event > image files. 

I have photos dating back to 2013 all the way through this month and It seems like PP only picked up through roughly 2014. If I go to the calendar section or the folders section, I see all of the folders, but if I click into them it says they are empty. 

I'm not using the import function and can't figure out why I only get some indexes. 

If I run the indexer again and map directly to a folder vs "All Originals" it seems to work. For example, rather than "All Originals" I will index /Nikon D610/2017 and that seems to work, but why wouldn't it pick this up automatically from the base folder? 

I'd rather not go through each folder and manually import and appreciate any help. 

Thanks,

**Comment by graciousgrey** [comment]
Is it possible that the indexing process was interrupted for some reason? Checking the logs while indexing "all originals" might give you some hints:

https://docs.photoprism.app/getting-started/troubleshooting/logs/

See also: https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

**Reply by zollan23** [reply]
I do not see any relevant errors in the logs and have gone through the troubleshooting for missing pictures. I can't seem to figure it out. The indexing process was not interrupted and I've done it multiple times and it still doesn't catch the other folders. For all intents and purposes we're talking about the exact same folder structure, camera and files... just different years and it's not catching it. 

Appreciate the help.

**Reply by lastzero** [reply]
Since you use a ppignore file, it may be caused by this. However, it was not provided by you so nobody is able to check it. Also note personalized support directly from our team is only available to sponsors.
